tests/compilation_tests/database_test.py

```diff
@@ -57,106 +32,89 @@ def test_str(self):
-                           "position_pid [proportional factor, integral factor, derivative factor]: [1, 0, 0],\n" \
+                           "position_pid [proportional factor, integral factor, derivative factor]: [0, 0, 0],\n" \
                            "position_noise: 0,\n" \
-                           "heading_pid [proportional factor, integral factor, derivative factor]: [1, 0, 0]"
-
-        testcases = [(db_str, self.db_default), (db_initial_str, self.db_initial), (db_one_param_str, self.db_one_param)]
+                           "heading_pid [proportional factor, integral factor, derivative factor]: [0, 0, 0]"
 
+        testcases = [(db_str, self.db), (db_initial_str, self.db_initial), (db_one_param_str, self.db_one_param)]
 
         for (expected_ans, database) in testcases:
-            self.assertEqual(expected_ans, str(database), str(database))
+            self.assertEqual(expected_ans, str(database))
 
     def test_get_data(self):
 
-        db_params = [(Phase.SETUP, "phase"), ([0, 0, 0], "state"), (True, "is_sim"), (0, "plastic_weight"),
-                     (100, "battery"), (0.5, "move_dist"), ([0, 0, 0], "acceleration"),
+        db_params = [(0, "phase"), ([0, 0, 0], "state"), (True, "is_sim"), (0, "plastic_weight"),
+                     (0, "battery"), (0.5, "move_dist"), ([0, 0, 0], "acceleration"),
                      ([0, 0, 0], "magnetic_field"), ([0, 0, 0], "gyro_rotation"),
-                     ([1, 0, 0], "position_pid"), (0, "position_noise"), ([1, 0, 0], "heading_pid")
+                     ([0, 0, 0], "position_pid"), (0, "position_noise"), ([0, 0, 0], "heading_pid")
                      ]
 
-        db_initial_params = [(Phase.TRAVERSE, "phase"), ([10, 20, 50], "state"), (False, "is_sim"),
+        db_initial_params = [(2, "phase"), ([10, 20, 50], "state"), (False, "is_sim"),
                              (2, "plastic_weight"),
                              (98, "battery"), (0.6, "move_dist"), ([4.25, 3.2, 0.1], "acceleration"),
                              ([0.1, 0.2, 0.3], "magnetic_field"), ([0.5, 0.2, 0.6], "gyro_rotation"),
                              ([0.12, 1.7, 9.2], "position_pid"), (0.23, "position_noise"),
                              ([0.2, 0.4, 0.16], "heading_pid")
                              ]
 
-        db_one_params = [(Phase.SETUP, "phase"), ([0, 0, 0], "state"), (True, "is_sim"), (3, "plastic_weight"),
+        db_one_params = [(0, "phase"), ([0, 0, 0], "state"), (True, "is_sim"), (3, "plastic_weight"),
                          (46, "battery"), (0.5, "move_dist"), ([0.5, 0.2, 0.3], "acceleration"),
                          ([0, 0, 0], "magnetic_field"), ([0, 0, 0], "gyro_rotation"),
-                         ([1, 0, 0], "position_pid"), (0, "position_noise"), ([1, 0, 0], "heading_pid")
+                         ([0, 0, 0], "position_pid"), (0, "position_noise"), ([0, 0, 0], "heading_pid")
                          ]
```

Comment:
why change phase back into a magic number? (Phase.SETUP into 0)

Reply:
outdated change -- should be updated with merge. 

<img width="845" alt="Screen Shot 2022-09-18 at 9 00 42 PM" src="https://user-images.githubusercontent.com/44074422/190935837-4dc8e9bc-0598-4be0-97df-d2961f2e61e5.png">